### DISCUSSION TITLE
Running vs walking

### DISCUSSION THREAD

**Discussion by nieko-nera** [cagnulein/qdomyos-zwift#1274] [Q&A]
Hello. How can i start not running, but walking activity in QZ?

**Comment by cagnulein** [comment]
@nieko-nera qz doesn't make difference about walking or running. Do you mean in strava? I guess there is no way inside QZ, you have to change it from strava directly (otherwise I have to add a new setting to specify this)

**Reply by nieko-nera** [reply]
Yes, in Strava. Maybe if workout averange speed is lower than 6 or 7 km/h mark it as walking?

**Reply by cagnulein** [reply]
yeah maybe an average speed under 6.5 could make the trick. Let me try to implement this
